web/modules/weather_data/src/Service/DataLayer.php

```diff
@@ -107,49 +107,49 @@ private function getFromWeatherAPI($url, $attempt = 1, $delay = 75)
         }
     }
 
-    private $i_alertsState = false;
+    private static $i_alertsState = false;
```

Comment:
I've been trying to look around to double check, but are the classes themselves initialized per-request?

Reply:
This one and WeatherDataService are constructed twice per request. It is not at all clear to me why that is, but once I saw it was happening, I figured static i-vars was a way around it. It seems to be working.